api.go

```diff
@@ -498,7 +499,7 @@ func NewRaft(conf *Config, fsm FSM, logs LogStore, stable StableStore, snaps Sna
 	// Create Raft struct.
 	r := &Raft{
 		protocolVersion:       protocolVersion,
-		applyCh:               make(chan *logFuture),
+		applyCh:               make(chan *logFuture, conf.ApplyChSize),
```

Comment:
Should this now just be `conf.MaxAppendEntries` given that it will at most pull that many off the channel.